src/backend/joanie/core/admin.py

```diff
@@ -313,16 +313,16 @@ def generate_certificates(self, request, queryset):  # pylint: disable=no-self-u
         certificate_generated_count = helpers.generate_certificates_for_orders(queryset)
         summarize_certification_to_user(request, certificate_generated_count)
 
-    def proforma_invoice(self, obj):  # pylint: disable=no-self-use
+    def invoice(self, obj):  # pylint: disable=no-self-use
         """Retrieve the root pro forma invoice related to the order."""
```

Comment:
I think you should search & replace `pro forma`
```suggestion
        """Retrieve the root forma invoice related to the order."""
```

Reply:
good catch!